server/views/pages/overview/partials/overviewTabContents.njk

```diff
@@ -1,97 +1,5 @@
-                  text: "Add a new goal",
-                  href: "/plan/" + prisonNumber + "/goals/create",
-                  id: "add-goal-button",
-                  classes: "moj-button-menu__item moj-page-header-actions__action"
-                }) }}
-
-              </div>
-            </div>
-          </div>
-
-        {% endif %}
-
-      </div>
-
-      {% for goal in actionPlan.goals %}
-        <div class="govuk-summary-card">
-          <div class="govuk-summary-card__title-wrapper">
-            <h2 class="govuk-summary-card__title">Goal {{ loop.index }}</h2>
-          </div>
-          <div class="govuk-summary-card__content">
-            <p class="govuk-body">{{ goal.title }}</p>
-            <p class="govuk-hint">Created on {{ goal.createdAt | formatDate('DD MMMM YYYY') }}
-              by {{ goal.createdByDisplayName }}</p>
-
-            <table class="govuk-table">
-              <caption class="govuk-table__caption govuk-visually-hidden">Goal {{ loop.index }} steps</caption>
-              <thead class="govuk-table__head">
-              <tr class="govuk-table__row">
-                <th scope="col" class="govuk-table__header">Steps</th>
-                <th scope="col" class="govuk-table__header">Target date</th>
-                <th scope="col" class="govuk-table__header">Status</th>
-              </tr>
-              </thead>
-              <tbody class="govuk-table__body">
-
-              {% for step in goal.steps %}
-
-                {% if step.status == "COMPLETE" %}
-                  {% set tagClass = "govuk-tag--green" %}
-                  {% elif step.status == "ACTIVE" %}
-                  {% set tagClass = "govuk-tag--blue" %}
-                {% else %}
-                  {% set tagClass = "govuk-tag--grey" %}
-                {% endif %}
-
-                <tr class="govuk-table__row">
-                  <td class="govuk-table__cell">{{ step.sequenceNumber }}. {{ step.title }}</td>
-                  <td class="govuk-table__cell">{{ step.targetDateRange | formatDateFormValue }}</td>
-                  <td class="govuk-table__cell">
-                    <strong class="govuk-tag {{ tagClass }}">
-                      {{ step.status | replace("_", " ") }}
-                    </strong>
-                  </td>
-                </tr>
-              {% endfor %}
-
-              </tbody>
-            </table>
-
-            {% if goal.note %}
-              <p class="govuk-body">Note: {{ goal.note }}</p>
-            {% endif %}
-
-          </div>
-        </div>
-
-      {% endfor %}
-
-    </div>
-
-  {% else %}
-    <div class="govuk-grid-column-two-thirds">
-      <h2 class="govuk-heading-m">Sorry, the service is currently unavailable.</h2>
-      <p class="govuk-body">
-        Retrieving goals is currently unavailable. Please try again later.
-      </p>
-    </div>
-  {% endif %}
-
+  <div class="govuk-grid-column-two-thirds">
+    {% include './overviewTabActionPlanColumn.njk' %}
+  </div>
```

Comment:
Include the new partial in the 2 thirds column

I'm imagining that @chrimesdev will be adding a new 1 third column for the Functional Skills stuff, so hopefully this is a neater / tidier way of doing it (otherwise this template will be massive)